libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -655,6 +657,7 @@ class SRV_Channels {
     AP_Int8 dshot_rate;
     AP_Int8 dshot_esc_type;
     AP_Int32 gpio_mask;
+    AP_Int16 rc_fs_mask;
```

Comment:
we should consider making this variable more closely match the parameter name so that it is easier for developers to find/remember.

Reply:
done